### ISSUE TITLE
[BUG]Timing trigger exception.

### ISSUE THREAD

**Issue by clown599** [elsa-workflows/elsa-core#5883]
## Description
Triggering the workflow every second via Cron or Timer can lead to repetitive workflow triggering and confusing timing.

## Steps to Reproduce
CRON(0/1 * * * * ?)->A time-consuming custom activity

## Expected Behavior
Triggered once per second according to the expression or after the first real column workflow has completed execution, then triggered the second workflow.

## Actual Behavior
A second or even more workflows are triggered during the execution of the first workflow instance

## Screenshots
![image](https://github.com/user-attachments/assets/c7fc5447-50af-4803-9dce-5e0c7923b958)

## Environment
- **Elsa Package Version**: 3.2.0-rc4
- **Operating System**:  Windows 10
- **Browser and Version**: Edge 127.0.2651.95

## Log Output
No error logging output

## Troubleshooting Attempts
When the execution time of a custom activity (which is a time-consuming activity) is controlled within the trigger interval or the trigger time is set beyond the activity execution time, the abnormal workflow real column will be greatly reduced or even no.

**Comment by clown599** [comment]
This setting does not appear to be working.
![image](https://github.com/user-attachments/assets/d1a1d4d4-a83a-49f7-8b35-ea92d66afc25)

**Comment by sfmskywalker** [comment]
@clown599 If I understand correctly, everything works as expected, except for the Singleton setting, is that right?

**Comment by clown599** [comment]
@sfmskywalker no. None of the above Settings work. And the trigger cycle is not expected.

**Comment by sfmskywalker** [comment]
Got it. So none of the Activation strategy settings work, and the cron trigger doesn't trigger every one second, but it does trigger regularly. 

- What is the average trigger interval, give or take? 
- What timer provider are you using? Quartz.NET, Hangfire, or the default one?

**Comment by clown599** [comment]
Yes.
1. There is no specific average trigger time. Sometimes it takes several seconds or even more than 10 seconds to trigger, and sometimes it triggers multiple times a second. The activity is triggered as expected only if the execution time is less than the trigger interval.
2, tried the default Settings and Quartz.NET, both have this problem.

Quartz.NET I set it up like this:
![image](https://github.com/user-attachments/assets/a953e11a-4594-4d92-bd4c-9671796a77f7)

**Comment by clown599** [comment]
@sfmskywalker  Click the following image to reproduce:
![image](https://github.com/user-attachments/assets/e77ddb0e-6f67-4b7b-beb7-174050d3347f)
![image](https://github.com/user-attachments/assets/d6ed6b13-efff-439a-a6d1-15765c9651f5)
![image](https://github.com/user-attachments/assets/e6be5dfc-c342-4361-bc1f-b9f1b179e521)
